recipe/meta.yaml

```diff
@@ -22,17 +22,17 @@ requirements:
     - ipython >=5.0
     - jupyter_client
     - pip
-    - python
+    - python >=3.4
   run:
     - ipython >=5.0
     - jupyter_client
-    - python
+    - python >=3.4
     - tornado >=4.2
     - traitlets >=4.1
 
 test:
   requires:
-    - curio  # [unix]
+    - curio
     - nose
     - numpy
     - pytest
```

Comment:
Yeah, pinning this to `pytest !=5.3.4` does get the `noarch` to build locally... haven't tried on windows.